libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -789,6 +789,13 @@ singleton AP_Winch method release_length void float'skip_check
 singleton AP_Winch method set_desired_rate void float'skip_check
 singleton AP_Winch method get_rate_max float
 
+include AP_Gripper/AP_Gripper.h
+singleton AP_Gripper depends AP_GRIPPER_ENABLED
+singleton AP_Gripper rename gripper
+singleton AP_Gripper method grab void
+singleton AP_Gripper method release void
+singleton AP_Gripper method released boolean
```

Comment:
There are locking issues with using the gripper from Lua. Releasing the gripper (with the same issues) is already possible with something like `rc:run_aux_function(19, 2)`. You could do an edge detect on the thrust loss instead of checking if the gripper is already released.

Can we use already existing functionality instead of adding a new problematic binding? We can come back once the issues get addressed.

Reply:
Hi @tpwrules, thanks for the feedback.

I'm not aware of the locking issues but regarding using edge detection, this also crossed my mind.  The way it is here it will override a pilot which is perhaps not a good thing but I'm not sure it really matters.  I don't think we expect this script to be used much so while I normally strive for perfection, I think it's good enough.

Reply:
Here is the background: https://github.com/ArduPilot/ardupilot/pull/27365#discussion_r1653164720

If it is a rarely used script then adding a potentially problematic binding just for it seems like an avoidable choice. I mentioned edge detection as a way to avoid needing a binding to check if the gripper is released or not.

Reply:
OK, switched to use the aux function as suggested, thanks!